### PR TITLE
Describe process to add new changes during a merge

### PR DESCRIPTION
**Pull request by athos-ribeiro**
Signed-off-by: Athos Ribeiro <athos.ribeiro@canonical.com>

### CODE REVIEW

**Review by basak — APPROVED** [review]
Looks great! A couple of minor comments inline.

File: PackageMerging.md [hunk 1]
```diff
@@ -440,6 +439,13 @@ Removing `1aed93f` will remove the patch.
     $ quilt pop -a
 
 
+### Adding new changes
+
+Add any new changes you may want to include with the merge. For instance, the new package version may be on a FTBFS state in Ubuntu due to new versions of specific libraries or runtimes.
```
Comment (utkarsh2102) [inline PackageMerging.md:444]:
```suggestion
Add any new changes you may want to include with the merge. For instance, the new package version may FTBFS in Ubuntu due to new versions of specific libraries or runtimes.
```

File: PackageMerging.md [hunk 2]
```diff
@@ -459,6 +465,14 @@ If you dropped any changes (due to upstream fixes), you must note them in the ch
         - Foo: change to bar
           [Fixed in 1.2.3-4]
 
+#### Format any new added changes
+
+If you added any new changes, they should be in their own section in the changelog:
+
+      * Added Changes:
+        - Bar: change to foo
+        - Biz: adjust for Foo changes
```
Comment (utkarsh2102) [inline PackageMerging.md:474]:
```suggestion
        - Baz: adjust for Foo changes
```
Wow, not Baz? Hahaha, this is just a very useless suggestion, I know. :)

File: PackageMerging.md [hunk 3]
```diff
@@ -459,6 +465,14 @@ If you dropped any changes (due to upstream fixes), you must note them in the ch
         - Foo: change to bar
           [Fixed in 1.2.3-4]
 
+#### Format any new added changes
+
+If you added any new changes, they should be in their own section in the changelog:
+
+      * Added Changes:
```
[the remark this answers is not in the document]
Comment (sergiodj) [reply PackageMerging.md:472]: I've only ever seen `Added changes` be used.  I don't remember seeing `Add changes`.  A quick `grep` in my workdir confirms this feeling:

```
$ grep -H '\* Add changes:' */*/debian/changelog
$ grep -H '\* Added changes:' */*/debian/changelog
adcli/adcli/debian/changelog:  * Added changes:
chrony/chrony/debian/changelog:  * Added changes:
chrony/chrony/debian/changelog:  * Added changes:
chrony/chrony/debian/changelog:  * Added changes:
net-snmp/net-snmp/debian/changelog:  * Added changes:
nss/nss/debian/changelog:  * Added changes:
nss/nss/debian/changelog:  * Added changes:
nut/nut/debian/changelog:  * Added changes:
openldap/openldap/debian/changelog:  * Added changes:
openvpn/openvpn/debian/changelog:  * Added changes:
python-rtslib-fb/python-rtslib-fb/debian/changelog:  * Added changes:
qemu/qemu/debian/changelog:  * Added changes:
spice/spice/debian/changelog:  * Added changes:
squid3/squid3/debian/changelog:  * Added changes:
squid/squid/debian/changelog:  * Added changes:
squid/squid/debian/changelog:  * Added changes:
squid/squid/debian/changelog:  * Added changes:
sssd/sssd/debian/changelog:  * Added changes:
strongswan/strongswan/debian/changelog:  * Added changes:
strongswan/strongswan/debian/changelog:  * Added changes:
strongswan/strongswan/debian/changelog:  * Added changes:
```
I understand that it is a good practice to write changelog entries (not only Ubuntu's or Debian's, but any changelog) using the imperative mood, and to be honest that's what I do when I write changelogs for Debian packages, but the team has been using the past tense specifically for those Added/Dropped bullets for a while now, so I don't see a problem in having this reflected in the manual.

**Review by sergiodj — APPROVED** [review]
LGTM, thanks!

**Comment by basak** [comment]
On Wed, Jan 26, 2022 at 10:20:54AM -0800, Sergio Durigan Junior wrote:
> I understand that it is a good practice to write changelog entries (not only Ubuntu's or Debian's, but any changelog) using the imperative mood, and to be honest that's what I do when I write changelogs for Debian packages, but the team has been using the past tense specifically for those Added/Dropped bullets for a while now, so I don't see a problem in having this reflected in the manual.

I think things have drifted away from best practice over time. It's
great that there is a manual now, because that can help stop the drift.
That means that we should use this opportunity to codify best practice
and adjust, not freeze and continue propagating the error.

File: PackageMerging.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bryceharrington) [reply PackageMerging.md:472]: I don't use 'Added changes' myself, but don't mark it as needing changed when doing reviews.  The main reason I avoid using it is for situations where there are multiple people contributing changes; in that case having each as its own bullet point with attribution seems to make for a cleaner changelog.  However, admittedly that situation doesn't come up very often.

Regarding tense, it does seem like 'Dropped changes' has become our team's convention, though I think I've seen other variations.

**Comment by athos-ribeiro** [comment]
Thanks for all the comments :)

I addressed all the straightforward ones.

For the section which refers to the changes being included with the merge, I changed the wording to "New Changes" as per Robie's alternative suggestion.

**Comment by bryceharrington** [comment]
Agreed, looks great!

**Pull request merged**